frontend/src/api/BaseApi.ts

```diff
@@ -18,44 +18,57 @@ export interface HeadersDict {
 }
 
 export default abstract class BaseApi {
-  /**
-   * Root path of API resource without leading slash.
-   */
+  // Root path of API resource without leading slash.
   abstract get basePath(): string;
 
-  /**
-   * Namespace representing the API resource.
-   */
+  // API version
+  get version() {
+    return "v0.1";
+  }
+
+  // Namespace representing the API resource
   abstract get namespace(): string;
```

Comment:
@chouinar this is kind of what I was thinking by making this extensible - we build the API URL as:

`basePath` + `version` + `namespace` + `subpath`

for local this would be => localhost:8080/0.1/opportunities/search